### PR TITLE
DOC-1986: Added TINY-9770 to the release notes.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-1986

Changes:
* Added TINY-9770 to the release notes.

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] ~`modules/ROOT/nav.adoc` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] ~Files removed have been deleted, not just excluded from the build (if applicable)~
- [x] ~(New product features only) Release Note added~

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 1]
```diff
@@ -30,6 +30,23 @@
...
+=== Advanced Templates 1.2.0
+
+The {productname} 6.5 release includes an accompanying release of the **Advanced Templates** premium plugin.
+
+**Advanced Templates** 1.2.0 includes one fix:
+
+=== The currently selected tree component template item would not stay highlighted after the Templates dialog redialing.
+//#TINY-9770
+
+In previous versions of {productname} a selected tree component template item would lose its highlight when the Advanced Template dialog was reopened.
```
Comment (vpyshnenko) [inline modules/ROOT/pages/6.5-release-notes.adoc:42]:
```suggestion
In previous versions of {productname} a selected tree component template item would lose its highlight when the Advanced Template dialog was rerendered.
```

File: modules/ROOT/pages/6.5-release-notes.adoc [hunk 2]
```diff
@@ -30,6 +30,23 @@
...
+
+This would occur when a user renamed a selected template item in the tree component. As a consequence, the user was unable to identify the currently selected item in the tree.
+
+In {productname} 6.5, the problem was addressed by utilizing the `defaultSelectedId` configuration property from the new tree component API.
+
+As a result, the previewed template now accurately reflects the selection in the tree component.
```
Comment (vpyshnenko) [inline modules/ROOT/pages/6.5-release-notes.adoc:48]:
```suggestion
As a result, the currently previewed template is now accurately highlighted in the tree component.
```

Comment (brian-forte) [inline modules/ROOT/pages/6.5-release-notes.adoc:48]:
```suggestion
=== A selected tree component template item did not stay highlighted when the Templates dialog re-rendered.
//#TINY-9770

In previous versions of {productname}, a selected tree component template item would lose its highlight when the Advanced Template dialog was re-rendered after the template list updated.

This occured when a user renamed a selected template item in the tree component. As a consequence, the user could not identify the currently selected item in the tree.

In {productname} 6.5, the problem was addressed by utilizing the new `defaultSelectedId` configuration property from the xref:dialog-components.adoc#tree[tree component] API.

As a result, the currently previewed template is now correctly highlighted in the tree component, as expected.

For information on the **Advanced Templates** plugin, see: xref:advanced-templates.adoc[Advanced Templates].
```
Copy-edits and some additions.

File: modules/ROOT/pages/6.5-release-notes.adoc (hunk 1 above)
Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc]: @vpyshnenko Applied @brian-forte copy edit changes to this.

File: modules/ROOT/pages/6.5-release-notes.adoc (hunk 2 above)
Comment (kemister85) [reply modules/ROOT/pages/6.5-release-notes.adoc]: @vpyshnenko Applied @brian-forte copy edit changes to this.

**Review by vpyshnenko — APPROVED** [review]

**Review by brian-forte — APPROVED** [review]

**Pull request merged**